src/Console/Mappings/IndexCopyCommand.php

```diff
@@ -35,51 +32,53 @@ public function handle()
     {
         ['from' => $fromIndex, 'to' => $toIndex] = $this->arguments();
 
-        $bulkParams = ['body' => []];
         $connection = DB::connection('elasticsearch');
 
-        $query = [
-            'index' => $fromIndex,
-            'size' => 0,
-            'body' => [
-                'query' => [
-                    'match_all' => (object) []
-                ],
+        $body = [
+            'source' => [
+                'index' => $fromIndex,
+            ],
+            'dest' => [
+                'index' => $toIndex,
             ],
         ];
 
-        $numResults = $connection->select($query)['hits']['total'];
-        $bar = $this->output->createProgressBar($numResults);
-        $bar->start();
+        try {
+            $this->output->write("Copying ${fromIndex} to ${toIndex}...", true);
```

Comment:
Think you've spent too much time writing Javascript :P
```suggestion
            $this->output->write("Copying {$fromIndex} to {$toIndex}...", true);
```